### DISCUSSION TITLE
[揭秘]前辈是可以吃东西的

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1473] [其他]
![Screenshot_20240811_155433](https://github.com/user-attachments/assets/68d8c620-1582-4f8e-92d1-2f6fb67f44c3)
晚上好，新艾利都，刚和前辈巡查完去六分街吃了碗面，瀑汤谷的叉烧面果然好吃啊。

顺带一提，前辈虽然是机器人，但其实是可以吃拉面的。但她说拉面吃起来太麻烦了还不如141杂货店的甜甜圈。

听说最近有大胃王免单活动，欢迎各位市民来品尝！

**Comment by ZhuYuan-148** [comment]
海鲜面也不错
![mmexport1723351278984](https://github.com/user-attachments/assets/984da39b-3d07-4691-9af6-0269b0f71796)

**Comment by gaojiaobei** [comment]
早就听闻鲤鱼面馆的大名，看这两张图，更是心向往之，可惜去的机会很少，真是实名羡慕警官你们

**Reply by ZhuYuan-148** [reply]
回头让前辈请你吃一碗。

**Comment by AnbulesPeo** [comment]
这个海鲜面。。。真的有点想吃了呢

**Reply by ZhuYuan-148** [reply]
吃完就感觉不累了。偶尔下趟馆子也不错

**Comment by SSTV-10068** [comment]
前辈吃东西为什么不张嘴啊，是在摆拍接广吗？SSTV有专门的广告专栏喔
[广告专栏页面](https://github.com/share121/inter-knot/discussions/1065)
![Uploading 111.jpg…]()

**Comment by keerdun** [comment]
青衣前辈可爱捏，想狠狠滴rua

**Reply by ZhuYuan-148** [reply]
前辈可没有那么可爱哦，她的年龄已经有………………

**Comment by hsxz-site** [comment]
这委托我接了

**Comment by DSSM1031** [comment]
朱鸢小姐  你带我走吧😭

**Reply by ZhuYuan-148** [reply]
可以呀，带你来治安局逛逛？（开个玩笑）

**Comment by xiaolvvvvvv** [comment]
机器人也要吃面吗

**Comment by Obsidia-cdl** [comment]
朱鸢前辈，那家店叫 瀑汤谷 不是 爆汤谷。然后那边除了叉烧面和海鲜面还有没有别的值得一试的面？

**Reply by ZhuYuan-148** [reply]
感谢后辈提醒已修改，打错字是我的失职。

其他面还没尝过，今天下班我请你一起吧。

**Comment by capture-sudo** [comment]
(╯▽╰ )好香~~！！！！kksk
